vowpalwabbit/vwdll.cpp

```diff
@@ -54,6 +54,14 @@ VW_DLL_MEMBER VW_HANDLE VW_CALLING_CONV VW_InitializeA(const char * pstrArgs)
   return static_cast<VW_HANDLE>(all);
 }
 
+VW_DLL_MEMBER VW_HANDLE VW_CALLING_CONV VW_SeedWithModel(VW_HANDLE handle, const char * extraArgs)
+{
+  string s(extraArgs);
+  vw* origmodel = static_cast<vw*>(handle);
```

Comment:
did you check how the C++ CLI layer (e.g. C# binding) calls this. I vaguely remember that there is some free() required.